docs/howto/index.md

```diff
@@ -26,5 +26,6 @@ Manage the charm version <manage-the-charm-version>
 Manage the workload version <manage-the-workload-version>
 Manage metrics <manage-metrics>
 Trace your charm <trace-your-charm>
+Make your charm discoverable <request-public-listing>
```

Comment:
The page title is quite different from the filename (perhaps they were the same at first). Should we make the filename `<mark-your-charm-discoverable>`?